codalab/lib/cli_util.py

```diff
@@ -71,11 +72,19 @@ def parse_key_target(spec):
         - <key>: (<key> if present,
                     empty string if ':' in spec but no <key>,
                     None otherwise)
-        - <value> (where value is everyhing after a <key>: (or everything if no key specified)
+        - <value> (where value is everything after a <key>: (or everything if no key specified)
     """
-
     match = re.match(TARGET_KEY_REGEX, spec)
-    return match.groups()
+    key, value = match.groups()
```

Comment:
Can you clarify here in a comment which part is `key` and which part is `value` of the entire target spec,

` [[<key>]:][<instance>::][<worksheet_spec>//]<bundle_spec>[/<subpath>]`?